rl/util.py

```diff
@@ -1,13 +1,19 @@
+import warnings
```

Comment:
I'm confused why this shows up in the diff since it is identical to the current state @ master. Doesn't really matter though.